model-engine/model_engine_server/inference/vllm/Dockerfile.vllm

```diff
@@ -0,0 +1,48 @@
+# syntax=docker/dockerfile:1
+ARG VLLM_VERSION=0.5.3.post1
+ARG VLLM_BASE_IMAGE=vllm/vllm-openai:v${VLLM_VERSION}
+FROM ${VLLM_BASE_IMAGE} AS base
+
+RUN apt-get update \
+    && apt-get install -y wget gdb psmisc dumb-init \
+    && apt-get autoremove -y \
+    && rm -rf /var/lib/apt/lists/* \
+    apt-get clean
+
+# Need to fix flashinfer at 0.0.8 to support gemma models
+# See https://github.com/vllm-project/vllm/issues/7060#issuecomment-2266248014
+#  vLLM 0.5.3 depends on torch 2.3.1
+RUN pip uninstall flashinfer -y
+RUN pip install flashinfer==0.0.8 --index-url https://flashinfer.ai/whl/cu121/torch2.3
+
+RUN wget https://github.com/peak/s5cmd/releases/download/v2.2.1/s5cmd_2.2.1_Linux-64bit.tar.gz
+RUN tar -xvzf s5cmd_2.2.1_Linux-64bit.tar.gz
+
+# symlink python to python3
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+FROM base AS vllm
+
+WORKDIR /workspace
+
+COPY model-engine/model_engine_server/inference/vllm/requirements.txt /workspace/requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
do we still end up needing the pip install requirements.txt? they're just vllm + pydantic at this point, so if we drop the vllm req from requirements then (as long as the pydantic install doesn't bust the python env) we can just omit having to make sure the VLLM_VERSION in the build script + vllm version in req.txt match

Reply:
true, i could remove it from the dockerfile; still want it around for local development